### ISSUE TITLE
Could you upload your tmux config?

### ISSUE THREAD

**Issue by aravezskinteeth** [dreamsofautonomy/dotfiles#5]
I've seen your videos, which have a Nvchad-like Tmux status bar. I'm not an expert on tmux, but could you share your tmux config in this repo so I can steal it?

**Comment by elliottminns** [comment]
Of course! I also have it available here: https://github.com/dreamsofcode-io/tmux

**Comment by diffficult** [comment]
quick question, is that the same config used in your last zen zsh video? You are doing some cool stuff with your status bar and the window tag numbers.

**Comment by pineapples5972** [comment]
You should video about tmux zen config

**Comment by elliottminns** [comment]
Absolutely!

My most up to date tmux config is at [github.com/elliottminns/dotfiles ](https://github.com/elliottminns/dotfiles/blob/main/.config/tmux/tmux.conf)

**Comment by kushvinth** [comment]
also the nix
